.gitignore

```diff
@@ -16,3 +16,7 @@ out/
 .vscode/*.log
 .gas-snapshot
 .DS_Store
+
+package-lock.json
```

Comment:
why would this end up in our foundry repo?

Reply:
I forcefully put prettier into the repo on my local machine. I can remove if needed, but I figured it's fine to add?

Reply:
there's a shortcut to format the code, isn't it working? SHIFT-OPTION-F on mac and something similar on windows

Reply:
Just had to reconfig to use the vscode Solidity plugin. Removed this.